### DISCUSSION TITLE
FingerPrintingSignalsProvider data is not showing

### DISCUSSION THREAD

**Discussion by sanjaysargam** [fingerprintjs/fingerprintjs-android#114] [Q&A]
```kotlin
val signals = withContext(context = Dispatchers.IO) {
                fingerprinter.getFingerprintingSignalsProvider()?.getSignalsMatching(
                    version = Fingerprinter.Version.V_5,
                    stabilityLevel = StabilityLevel.STABLE
                ).orEmpty()
            }
``` 
I called this function in app

### Actual Behaviour

[com.fingerprintjs.android.fingerprint.fingerprinting_signals.ManufacturerNameSignal@df64b7b, com.fingerprintjs.android.fingerprint.fingerprinting_signals.ModelNameSignal@8ffcd98, com.fingerprintjs.android.fingerprint.fingerprinting_signals.TotalRamSignal@11f09f1, com.fingerprintjs.android.fingerprint.fingerprinting_signals.TotalInternalStorageSpaceSignal@a4d9d6, com.fingerprintjs.android.fingerprint.fingerprinting_signals.ProcCpuInfoV2Signal@2142457, com.fingerprintjs.android.fingerprint.fingerprinting_signals.SensorsSignal@7f02b44, com.fingerprintjs.android.fingerprint.fingerprinting_signals.InputDevicesV2Signal@909542d, com.fingerprintjs.android.fingerprint.fingerprinting_signals.BatteryFullCapacitySignal@86cbd62, com.fingerprintjs.android.fingerprint.fingerprinting_signals.CameraListSignal@3ccaef3, com.fingerprintjs.android.fingerprint.fingerprinting_signals.GlesVersionSignal@e66b7b0, com.fingerprintjs.android.fingerprint.fingerprinting_signals.AbiTypeSignal@fd40629, com.fingerprintjs.android.fingerprint.fingerprinting_signals.CoresCountSignal@3dd4dae]

### Expected Behaviour

[FingerprintItemData(signalName=Manufacturer Name, signalValue=realme, stabilityLevel=STABLE, versionStart=V_1, versionEnd=V_5), FingerprintItemData(signalName=Model Name, signalValue=RMX3521, stabilityLevel=STABLE, versionStart=V_1, versionEnd=V_5), FingerprintItemData(signalName=Total Ram, signalValue=5879816192, stabilityLevel=STABLE, versionStart=V_1, versionEnd=V_5), FingerprintItemData(signalName=Total Internal Storage Space, signalValue=709652480, stabilityLevel=STABLE, versionStart=V_1, versionEnd=V_5), FingerprintItemData(signalName=ProcCpuInfo V2, signalValue=Hardware : Qualcomm Technologies, Inc SM6225

**Comment by Sergey-Makarov** [comment]
Hey Sanjay! I don't think I understand your question just yet, could you please clarify it a bit more? Actual vs expected behaviour is not self-explanatory to me.

**Reply by sanjaysargam** [reply]
I somehow got the desired data which I want. Thanks You

**Reply by Sergey-Makarov** [reply]
Alright, closing this discussion then :)
